.circleci/config.yml

```diff
@@ -63,13 +50,51 @@ jobs:
               python integration.py compare \
                 --report-path test-reports \
                 $upstream_short_id $head_short_id
-      - store_artifacts:
+      - store_artifacts: &store_artifacts
           path: /app/test-reports
           destination: /app/test-results
+      - save_cache: &save_cache
+          paths:
+            - ~/resources
+          key: mps-integration-data-v1-{{ epoch }}
+  integrate-clean:
+    docker:
+      - image: mozilla/edge-validator:1.3
+    steps:
+      - checkout
+      - run:
+          <<: *checkout_upstream
+      - setup_remote_docker
+      # We skip the restore_cache step so that the most recent cache is pristine once every day.
+      - run:
+          <<: *run_comparison
+      - store_artifacts:
+          <<: *store_artifacts
+      - save_cache:
+          <<: *save_cache
 
 workflows:
     version: 2
     build:
-        jobs:
-            - test
-            - integrate
+      jobs:
+        - test:
+            filters:
+              branches:
+                # This branch is only used for the .github/push-to-trigger-integration script
+                ignore: trigger-integration
+        - integrate:
+            filters:
+              branches:
+                # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
+                ignore: /pull\/[0-9]+/
+    nightly:
+      jobs:
+        - integrate-clean
+      triggers:
+        - schedule:
+            # Run a little after midnight Pacific every day.
+            cron: "33 8 * * *"
```

Comment:
This schedules the integration tests every night? If so, this is nifty.

Reply:
That is correct!